migrations/migrations/2_grant_priv_to_user.down.sql

```diff
@@ -0,0 +1 @@
+REVOKE SELECT, INSERT, UPDATE, DELETE ON filters FROM fuser;
```

Comment:
Is there any sense to use this?

Reply:
This is a part of issue "down migrations"